### PR TITLE
Mention Security Embargo Policy.

### PR DESCRIPTION
**Pull request by Shubham82**
Added Security Embargo Policy in Release Information docs.

Fixes: #28786

### CODE REVIEW

**Comment by Shubham82** [comment]
Hi @Verolop @zparnold 
could please take a look

**Review by sftim — COMMENTED** [review]
As this is just one paragraph, I wonder if it merits its own page. I'd aim to include that information in another document in the same folder (make sure not to update any that are generated).

_If  you do want to modify a generated page, you should instead find its source content and update that._

**Comment by Shubham82** [comment]
Thanks, @sftim for looking at this.
which document you want me to add this information. what do you suggest?

**Comment by Shubham82** [comment]
Hi @sftim, 
Is it legit to write in [Release Managers](https://kubernetes.io/releases/release-managers/)? what do you think?

**Comment by Shubham82** [comment]
Hi @sftim
Please take a look at this and please suggest your thoughts on this.

**Comment by sftim** [comment]
https://kubernetes.io/releases/release-managers/ seems like a good choice. (SIG Release would have the final say, I think).

**Comment by Shubham82** [comment]
Thanks @sftim

**Comment by saschagrunert** [comment]
> https://kubernetes.io/releases/release-managers/ seems like a good choice. (SIG Release would have the final say, I think).

Sounds good to me :+1:

**Comment by Shubham82** [comment]
Thanks @saschagrunert for looking at this.
One more question where in docs should I add this:
Should I add this before SIG Release Leads or before SIG Release Leads? what do you suggest?

**Comment by saschagrunert** [comment]
> Thanks @saschagrunert for looking at this.
> One more question where in docs should I add this:
> Should I add this before SIG Release Leads or before SIG Release Leads? what do you suggest?

How about a new subsection within "Contact": https://kubernetes.io/releases/release-managers/#contact

**Comment by Shubham82** [comment]
Sound good, +1

> How about a new subsection within "Contact": https://kubernetes.io/releases/release-managers/#contact

thanks, will soon create a PR for it.

**Comment by Shubham82** [comment]
Hi @saschagrunert @sftim  
Addressed the comment Please take a look.

**Review by saschagrunert — APPROVED** [review]
/lgtm

File: content/en/releases/release-managers.md
Comment (saschagrunert) [inline content/en/releases/release-managers.md:34]: non blocking nit, but we could probably omit the word "here":
```suggestion
Some information about releases is subject to embargo and we have defined policy about how those embargos are set. Please refer [Security Embargo Policy](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy) for more information.
```

**Pull request merged**
